### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by MAKWritingHouse**
config:
- eslint
- prettier
- jest
- APIs
handle APIs from next apis

### CODE REVIEW

**Review by makstyle119 — DISMISSED** [review]
Refactor

File: package-lock.json
Comment (MAKDevelopmentHouse) [inline package-lock.json:1]: remove `package-lock.json`

**Review by MAKDevelopmentHouse — CHANGES_REQUESTED** [review]
code is not documented

**Review by makstyle119 — DISMISSED** [review]
Refactor

**Review by MAKDevelopmentHouse — DISMISSED** [review]
Refactor

**Review by MAKDevelopmentHouse — APPROVED** [review]
**approved!**

**Review by makstyle119 — APPROVED** [review]
**Approved !**

**Pull request merged**